__test__/requirementsIntegration.test.tsx

```diff
@@ -50,7 +50,8 @@ test("Test whether requirements visually update on the screen", async () => {
   const openButton = screen.getByTestId("openDrawer");
   await user.click(openButton);
   let reqPercentages = screen.getAllByTestId("reqPercent");
-  expect(reqPercentages[1]).toContainHTML("9.1");
+  expect(reqPercentages[0]).toContainHTML("3.3");
```

Comment:
use the following to check by name, rather than the index:

```js
expect(within(parentEl(screen.getByText(/CS MA/i), "Requirement")).getByText(/44/i)).toBeInTheDocument();
```

This might rely on changes I've made in my branch, but I can send you the changes.

Reply:
I wasn't the one who originally wrote this test so I just followed the stylization from the file itself. I am thinking it should be fine for now but future tests should probably be doing what you suggested?